web3/trace.py

```diff
@@ -0,0 +1,77 @@
+from web3 import formatters
+from web3.utils.empty import (
+    empty,
+)
+
+class Trace(object):
+    def __init__(self, web3):
+        self.web3 = web3
+
```

Comment:
Using `class Trace(Module):` means that the `__init__` method can be deleted. Yay for less code! (requires importing `from web3.module import Module`)